### PR TITLE
:zap: Improve vehicle properties

### PR DESCRIPTION
**Pull request by spacevx**
extras id are between 1 and 15, i warned in the code that some native like IsToggleModOn() can return a boolean or a integer (if true going to return 1 otherwise going to return false)

I'm checking if the player who is in the vehicle is the owner of the entity (because it won't set the vehicle properties if not) and i'm using _NCS:Trace() to notify problems

### CODE REVIEW

File: src/api/vehicles/client/setVehicleProperties.lua [hunk 1]
```diff
@@ -4,213 +4,214 @@
 ---@return void
 ---@public
 function _NCS:setVehicleProperties(vehicleEntity, props)
-    if DoesEntityExist(vehicleEntity) then
-        local colorPrimary, colorSecondary = GetVehicleColours(vehicleEntity)
-        local pearlescentColor, wheelColor = GetVehicleExtraColours(vehicleEntity)
-        SetVehicleModKit(vehicleEntity, 0)
+    if not DoesEntityExist(vehicleEntity) then return _NCS:trace("Can't set vehicle properties for the vehicle (entity doesn't exist)") end
+    if NetworkGetEntityOwner(vehicleEntity) ~= PlayerId() then return _NCS:trace("Can't set vehicle properties for the vehicle (client is not the entity owner") end
```
Comment (PABLO-1610) [inline src/api/vehicles/client/setVehicleProperties.lua:8]: Please consider using _NCS#die introduced in this commit https://github.com/NextCitizens/ncs_core/commit/53bbdf825e1530fd24e0207bf7bdfcb1179f1a5c

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Just a little change

File: src/api/vehicles/client/setVehicleProperties.lua (hunk 1 above)
Comment (TheSpaceGamerV2) [reply src/api/vehicles/client/setVehicleProperties.lua]: Should be good now

**Review by PABLO-1610 — APPROVED** [review]
Thanks a lot 😁

**Pull request merged**